integration

```diff
@@ -290,6 +290,7 @@
   "cavefire/hass-openid",
   "cazeaux/ha-iracing",
   "cdnninja/yoto_ha",
+  "cedricziel/ha-matter-binding-helper",
```

Comment:
Your README states that telemetry is optional.
However, that is being enabled by default, while true, this is a pattern that will surprise users.

a) Change it to False by default.
b) Adjust documentation to state that this is enabled by default and you can disable it in the config options.

https://github.com/cedricziel/ha-matter-binding-helper/blob/v0.23.9/custom_components/matter_binding_helper/const.py#L9

Reply:
Appreciate you checking, @ludeeus  - when a user installs the integration, they're asked whether they want to share and can opt out. - I would consider this to be very explicit and more than you suggested.

<img width="545" height="495" alt="Bildschirmfoto 2025-12-16 um 16 57 04" src="https://github.com/user-attachments/assets/3377c460-b873-4db6-b2f4-e1af2d0ecf2a" />